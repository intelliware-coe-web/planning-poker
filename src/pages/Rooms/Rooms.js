import React from 'react';
import './Rooms.scss'
import Button from "../../components/Button";

function Rooms() {
    return (
        <div className="rooms">
            <p className="test">Choose a Room!</p>
            <Button name={'A'}/>
            <Button name={'B'}/>
            <Button name={'C'}/>
            <Button name={'D'}/>
            <Button name={'E'}/>
            <Button name={'F'}/>
            <Button name={'G'}/>
            <Button name={'+'}/>
        </div>
    );
}

export default Rooms;