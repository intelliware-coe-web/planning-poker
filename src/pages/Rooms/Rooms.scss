.rooms {
  margin-top: 15px;
}
