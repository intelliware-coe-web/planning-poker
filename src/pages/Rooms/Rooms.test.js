import React from 'react';
import Rooms from './Rooms';

it('rooms should match snapshot', () => {
    expect(Rooms()).toMatchSnapshot();
});