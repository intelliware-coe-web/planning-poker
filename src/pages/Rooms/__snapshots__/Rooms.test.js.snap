// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rooms should match snapshot 1`] = `
<div
  className="rooms"
>
  <p
    className="test"
  >
    Choose a Room!
  </p>
  <Button
    name="A"
  />
  <Button
    name="B"
  />
  <Button
    name="C"
  />
  <Button
    name="D"
  />
  <Button
    name="E"
  />
  <Button
    name="F"
  />
  <Button
    name="G"
  />
  <Button
    name="+"
  />
</div>
`;
